timemachine/fe/system.py

```diff
@@ -130,16 +120,16 @@ class VacuumSystem(Generic[_Nonbonded, _HarmonicAngle]):
     # utility system container
     bond: BoundPotential[HarmonicBond]
     angle: BoundPotential[_HarmonicAngle]
-    torsion: Optional[BoundPotential[PeriodicTorsion]]
+    proper: BoundPotential[PeriodicTorsion]
+    improper: BoundPotential[PeriodicTorsion]
     nonbonded: BoundPotential[_Nonbonded]
-    chiral_atom: Optional[BoundPotential[ChiralAtomRestraint]]
-    chiral_bond: Optional[BoundPotential[ChiralBondRestraint]]
```

Comment:
Nice, making these required will let us remove a lot of assertions in the code. Not recalling the original motivation for making them optional.